tools/release/build-mobilecoind-release.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
```

Comment:
mobildcoind releases are not related to the mirror and do not belong here.

Reply:
Removed in #7